src/lib/SortedMerkleTree.ts

```diff
@@ -96,12 +97,20 @@ export class SortedMerkleTree {
     return this.generateProof(0, this.getIndexForKey(key));
   }
 
-  getRoot(): string {
-    if (this.tree.length === 0) {
-      return emptyRoot;
-    }
+  getRoot(timestamp?: number): string {
+    const root = this.tree.length === 0 ? emptyRoot : this.tree[this.tree.length - 1][0];
+
+    return timestamp === undefined ? root : this.squashRoot(root, timestamp);
+  }
+
+  squashRoot(root: string, timestamp: number): string {
+    return `${root.slice(0, SortedMerkleTree.SQUASHED_ROOT_SIZE)}${timestamp
+      .toString(16)
+      .padStart(64 - SortedMerkleTree.SQUASHED_ROOT_SIZE, '0')}`;
```

Comment:
Can this 64 be hardcoded here? Is it the original root size?

Reply:
its size of `bytes32` type, it will never change, its ok to be hardcoded.